src/vs/workbench/contrib/notebook/browser/controller/chat/notebookChatController.ts

```diff
@@ -428,7 +423,7 @@ export class NotebookChatController extends Disposable implements INotebookEdito
 		this._notebookEditor.changeViewZones(accessor => {
 			const notebookViewZone = {
 				afterModelPosition: index,
-				heightInPx: 80 + WIDGET_MARGIN_BOTTOM,
```

Comment:
was this widget_margin_bottom causing the UI glitch?

Reply:
The inline chat widget missing a layout is causing the glitches horizontally, and this one is causing the vertical one ;(

Reply:
I seee, that makes sense.